### PR TITLE
Installing Helm and adding tests: CY-5792

### PR DESCRIPTION
**Pull request by stefanvacareanu7**

### CODE REVIEW

**Comment by h314to** [comment]
I think you forgot to add `.idea/` and `.DS_Store to `.gitignore`

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (lolgab) [inline .idea/.gitignore]: You don't need this file if you gitignore the entire directory.

**Comment by lolgab** [comment]
You can remove the files you committed by mistake, by running:
```
`git rm --cached .idea`
```

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,10 @@
 FROM python:3.9-alpine
 WORKDIR /workdir
 COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt && \
+    apk update && \
+    apk add --no-cache curl bash openssl; curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3; chmod 700 get_helm.sh; /bin/bash get_helm.sh; rm get_helm.sh
```
Comment (lolgab) [inline Dockerfile]: You can install helm from the Alpine packages: https://pkgs.alpinelinux.org/packages?name=helm&branch=edge

**Review by lolgab — CHANGES_REQUESTED** [review]

**Comment by stefanvacareanu7** [comment]
outdated

**Pull request closed without merging**